services/api/src/app.py

```diff
@@ -23,12 +26,15 @@
 
 from nnsight.schema.response import ResponseModel
 
-from .api_key import api_key_auth
 from .logging import load_logger
-from .metrics import RequestStatusMetric, TransportLatencyMetric
+
+logger = load_logger(service_name="API", logger_name="API")
```

Comment:
So does this get rid of the informative logs from the unicorn workers as they spin up? Those are pretty useful for determining whether the api is ready to receive requests